jenkins.sh

```diff
@@ -10,7 +10,7 @@ if [ -z "$CONJUR_CONTAINER" ]; then
 	    docker pull $DOCKER_IMAGE
 	fi
 	
-	cid=$(docker run -d -v ${PWD}:/src/conjur-api $DOCKER_IMAGE)
+	cid=$(docker run --privileged -d -v ${PWD}:/src/conjur-api $DOCKER_IMAGE)
```

Comment:
When I try to run without `--privileged`, I get this:
```
Alans-MacBook-Pro:conjur ajp$ docker -v
Docker version 1.12.0-rc2, build 906eacd, experimental
Alans-MacBook-Pro:conjur ajp$ docker run -i registry.tld/conjur-appliance-cuke-master:4.8-stable
keyctl_join_session_keyring: Operation not permitted
```
The alternative is to use a seccomp profile, like the one Rafal created: https://github.com/conjurinc/appliance/blob/master/distrib/seccomp.json, or disable it altogether with `--security-opt seccomp:unconfined`.

Reading a little more about `--privileged`, it's probably too big a hammer here. Given that this is just for running tests, though, maybe I'll just switch it to `--security-opt seccomp:unconfined`.